contracts/AuctionBase.sol

```diff
@@ -7,14 +7,20 @@ import "./utils/Pausable.sol";
 /// @dev Contains models, variables, and internal methods for the auction.
 contract AuctionBase is Pausable {
 
+  // Ongoing: Auction is accepting bids and is not cancelled.
+  // Cancelled: The seller cancelled the auction.
+  // Concluded: Winning bid decided, but NFT and balance transfers are yet to happen.
+  enum AuctionStatus { Ongoing, Cancelled, Concluded }
```

Comment:
Would prefer `{ Active, Cancelled, Completed }` if you don't feel strongly

Reply:
Cool